astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -30,6 +28,97 @@
 # prevent having to create new Angle objects every time `get_roll0` is called.
 _ROLL0 = Angle(58.5986320306*u.degree)
 
+
+class default_sun_galactocentric(ScienceState):
```

Comment:
I think I'd give a bit more explicit name, like `galactocentric_frame_[definition|parameters]` or so, in analogy with things we already have, like `solar_system_ephemeris` and `earth_rotation_table`.

Reply:
I opted for `galactocentric_frame_defaults` to try to make it clear that these only control the default parameters, i.e. I don't want users to mistakenly think that this is the way to generically set or change the frame parameters.

Reply:
Certainly fine (though note that `solarsystem_ephemeris` is similarly a default - it can be overridden in most routines where it is used)